dashy/src/app/dashboard/widget/jenkins/jenkins-widget.component.css

```diff
@@ -0,0 +1,74 @@
+.widget {
+  position: relative;
```

Comment:
You should move the css changes to `dashy/src/styles.css` so we don't have to apply them to every widget specific css file.